config/profile/rhdh/default-config/app-config.yaml

```diff
@@ -12,6 +12,10 @@ data:
               subject: legacy-default-config
               # This is a default value, which you should change by providing your own app-config
               secret: "pl4s3Ch4ng3M3"
+    auth:
+      providers:
+        github: {}
```

Comment:
I guess we can only have `auth.providers: {}` (without `github`) as the actual provider should not matter, according to the error message. It will also be consistent with the Helm Chart: https://github.com/redhat-developer/rhdh-chart/blob/main/charts/backstage/values.yaml#L53-L54

Reply:
I've gone ahead and pushed this change to unblock the failing nightly checks: https://github.com/redhat-developer/rhdh-operator/actions/workflows/nightly.yaml